publication-commons/src/main/java/no/unit/nva/publication/model/business/TicketEntry.java

```diff
@@ -275,6 +279,16 @@ private static boolean isPublished(Publication publication) {
         return PUBLISHED_STATUSES.contains(publication.getStatus());
     }
 
+    public abstract void validateAssigneeRequirements(Publication publication);
```

Comment:
Is this method meant to do some validation in the future?

Reply:
Yes, I thought that in case we need to validate ticket assignee in the future.